assets/css/_map_page.scss

```diff
@@ -133,6 +133,24 @@ $map-page-search-stub-size: 10px;
   padding: 0.5rem;
 }
 
+.c-map-page__search_results_section {
+  margin-bottom: 0.5rem;
+}
+
+.c-map-page__search_results_header {
+  color: $color-gray-900;
+  font-feature-settings: "clig" off, "liga" off;
```

Comment:
**question (non-blocking):** where does this come from? I can't seem to find anything in the designs about it, and it doesn't look like we've used `font-feature-settings` previously. No objections or anything, just curious!

Reply:
I pulled it from the designs [here](https://www.figma.com/file/sGUlB5cVN61RnXAjUm2JV2/Skate-%7C-Maps-%7C-Geolocation-Search?node-id=25%3A1442&mode=dev) - I *think* this should link directly to the right node, but let me know if that doesn't work!

Reply:
Ahh, okay, I was looking at the equivalent node in a separate part of the Figma document.